### DISCUSSION TITLE
GitHub campus pack

### DISCUSSION THREAD

**Discussion by vandana123maddi** [community/community#46562] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

Hey few days back i kept to GitHub pack program today i got mail that i was rejected saying that 
1)you are significantly more likely to be verified if you have completed your GitHub user profile with your full name and short bio.
I got this and updated my full name and bio
2)u appear not to be near any campus location for the school you selected .if your a distance learner then your school affiliation must state so !
I didn't got this point i am just 20min and 7km far from my college any one can say at what bases they say ur near or not i want to reapply  what should I do to verified these time

**Comment by jge162** [comment, marked as answer]
Just update your profile with a short bio and provide your school Id or transcripts showing your attending classes this semester. Good luck.

**Reply by vandana123maddi** [reply]
I provided my identity card before also then  why it got rejected then?

**Reply by jge162** [reply]
Your school id?

**Reply by vandana123maddi** [reply]
Not school college i provided photo

**Reply by jge162** [reply]
Not sure what you are saying. You provided a picture of your college id

**Reply by vandana123maddi** [reply]
I mean to say i am a college student not a school student 
And i provided  my college identity card as a proof

**Reply by jge162** [reply]
I understand. I don’t know why they are rejecting you then. You can try Submitting a GitHub support ticket. Her is the link https://support.github.com/request/education?tags=hubberfy_education

**Reply by vandana123maddi** [reply]
Thank u i will try  again 😇 have a good day 👍

**Reply by vandana123maddi** [reply]
i got githubstudent pack now  i think before i did not got because i  do not upload my student mail id
